samples/deployment/test-network/installFPC.sh

```diff
@@ -23,8 +23,12 @@ PEERS=("peer0.org1.example.com" "peer0.org2.example.com")
 ERCC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 ECC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 
-CC_ID="echo"
-CC_VER="$(cat ${FPC_PATH}/samples/chaincode/${CC_ID}/_build/lib/mrenclave)"
+if [[ -z "${TEST_CC_ID}" ]]; then
+  echo "TEST_CC_ID not set - using echo chaincode"
+  TEST_CC_ID=echo
+#  exit 1
```

Comment:
I'd rather make it fail here: no cc id specified... error!
Otherwise it would produce default actions which are unintended from the user/developer perspective.